examples/fabric/cloud_router/example_1/java/src/main/java/generated_program/App.java

```diff
@@ -3,8 +3,8 @@
 import com.pulumi.Context;
 import com.pulumi.Pulumi;
 import com.pulumi.core.Output;
-import com.pulumi.equinix.fabric.CloudRouter;
-import com.pulumi.equinix.fabric.CloudRouterArgs;
+import com.equinix.equinix.fabric.CloudRouter;
```

Comment:
As with docs, the old and new packages here are both wrong, but it might be better to restore the old wrong value to keep the diff small and then fix it later.